lib/ui/semantics.dart

```diff
@@ -300,6 +300,7 @@ class SemanticsFlag {
   static const int _kIsReadOnlyIndex = 1 << 20;
   static const int _kIsFocusableIndex = 1 << 21;
   static const int _kIsLinkIndex = 1 << 22;
+  static const int _kIsSliderIndex = 1 << 23;
```

Comment:
I think we forgot to mention, we also need to update `flutter/lib/web_ui/lib/src/ui/semantics.dart`

Can you also modify the comment here to also mention this file?

Reply:
as well as /Users/chtai/git/engine/src/flutter/shell/platform/android/io/flutter/view/AccessibilityBridge.java line 1748

Reply:
Sure, done.